### DISCUSSION TITLE
IExpressionBuilder.GetMapExpression<TSource, TDest>

### DISCUSSION THREAD

**Discussion by candoumbe** [LuckyPennySoftware/AutoMapper#3847] [NA]
I've been using AutoMapper since 7.x and I must say it's been integrated so smootlhy.
I recently tried to migrate from `10.x` to `11.x` and the only thing that stood out so far is `IExpresssionBuilder` that seems to be missing.

Is there a reason behind removing this interface ?

I've been using the `IExpressionBuilder.GetMapExpression<TSource, TDest>()` extension method quite a lot and would like to know if there is a replacement for this in the `11.x` version of the package ? This method has helped me in so many projects where I was able to dynamically retrieve the expression used to map `TSource`  to `TDest` (outside of any `IQueryable<T>` stuff) and use that expression however I wanted.

I have a small class demonstrating the purpose of this [here](https://github.com/candoumbe/MedEasy/blob/develop/src/tools/MedEasy.CQRS.Core/Handlers/HandleSearchQuery.cs)

Thank you in advance !

**Comment by lbargaoanu** [comment]
Renamed to `IProjectionBuilder`.
https://github.com/AutoMapper/AutoMapper/search?q=GetMapExpression
